UNRELEASED.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+**Added:**
```

Comment:
Just a reminder this should be breaking if we do not change the rule to disable theming if props are missing